XenAdmin/Dialogs/VMProtectionRecovery/PolicyHistory.cs

```diff
@@ -168,30 +184,30 @@ private void comboBox1_SelectedIndexChanged(object sender, EventArgs e)
         {
             if (_policy != null)

```

Comment:
All the code in this function can be reused in RefreshTab(), for example by extracting it into a function StartRefreshGrid() and calling this function in here and in RefreshTab() at line 86.